hangman.js

```diff
@@ -1 +1,49 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+
+var wordEntered = prompt("Please enter a word");
+var splitWord = wordEntered.split("");
+var wrongGuesses = [];
+
+var concealWord = function (x) {
+    var wordWithHints = [];
+    for (i = 0; i < x.length; i++) {
+        wordWithHints[i] = "-";
+    }
+    return wordWithHints;
+};
+var blankWord = concealWord(splitWord);
+
+var askForGuess = function() {
+  userGuess = prompt("Guess a letter.");
+  checkGuess();
+};
+
+
+var checkGuess = function() {
+    if (splitWord.includes(userGuess)) {
+        for (var i = 0; i < blankWord.length; i++) {
+          if (splitWord[i] === userGuess){
+            blankWord[i] = userGuess;
+          }
```

Comment:
I think you can move some of this logic to the `concealWord` function, but i like this solution!